catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java

```diff
@@ -338,6 +338,14 @@
                 true /* tokenized */,
                 false /* multivalued */,
                 BasicTypes.STRING_TYPE));
+
+        /* Experimental Attributes, subject to change*/
+        DESCRIPTORS.add(new AttributeDescriptorImpl(DATA_QUALITY,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
```

Comment:
Data Quality should be multivalued.  In MGMP, there are various types of fields that define data quality under the Resource Quality element (e.g. DQ_GriddedDataPositionalAccuracy, DQ_AbsoluteExternalPositionalAccuracy, etc).